docs/source/user_guide/concepts/observation.md

```diff
@@ -92,6 +92,71 @@ For a quick demo to visualize pointclouds, you can run
 python -m mani_skill.examples.demo_vis_pcd -e "PushCube-v1"
 ```
 
+### voxel
+This observation mode has the same data format as the [sensor_data mode](#sensor_data), but all sensor data from cameras are removed and instead a new key is added called `voxel_grid`.
+
+To use this observation mode, a dictionary of observation config parameters is required to be passed in via obs_mode_config during environment initializations (gym.make()). It should contain the following voxelization config hyperparameters:
```

Comment:
can you include documentation at the top of the document discussing about how some observation modes permit observation configurations (to override any defaults)? And how to do so?

For some tasks we might have some default voxel grid configs (to support easy benchmarking of methods like peract by simply doing gym.make(env_id) instead of having to have the user specify some set of voxel configs).